types/ApiResponses.ts

```diff
@@ -39,6 +40,14 @@ export interface SubmitAttendanceForUsersResponse extends ApiResponse {
   attendances: PublicAttendance[];
 }
 
+export interface ModifyUserAccessLevelResponse extends ApiResponse {
+  updatedUsers: PrivateProfile[];
+}
+
+export interface GetAllUserAccessLevelsResponse extends ApiResponse {
+  users: Partial<UserModel>[];
```

Comment:
@dowhep I had to import this type, should this be fine?

Reply:
Lets make a separate class explicitly defining what the access level response should contain like
```
interface AccessLevel {
    uuid: string,
    accessType: string
}
```
^ might not be entirely accurate to the way u implemented